temporalio/workflow.py

```diff
@@ -1220,6 +1236,7 @@ def __init__(
         super().__init__(logger, extra or {})
         self.workflow_info_on_message = True
         self.workflow_info_on_extra = True
+        self.update_info_on_extra = True
```

Comment:
Not sure this deserves a separate setting. I think these two pieces of update information can be considered workflow information (and we are considering them workflow information for purposes of appending to the message, might as well in the extra too consistently).